exchange/utils.go

```diff
@@ -40,11 +40,11 @@ func cleanOpenRTBRequests(ctx context.Context,
 	req AuctionRequest,
 	requestExt *openrtb_ext.ExtRequest,
 	bidderToSyncerKey map[string]string,
-	gDPR gdpr.Permissions,
+	vendorListFetcher gdpr.VendorListFetcher,
 	metricsEngine metrics.MetricsEngine,
 	gdprDefaultValue gdpr.Signal,
 	privacyConfig config.Privacy,
-	account *config.Account) (allowedBidderRequests []BidderRequest, privacyLabels metrics.PrivacyLabels, errs []error) {
```

Comment:
`account` was redundant here; it's already available in `req AuctionRequest` above.